training/models/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""Helper functions to store and get the selected model"""
+from collections import namedtuple
+from training.utils import named_module
+
+_current = None
+AVAILABLE = ['cnn', 'mlp']
+Model = namedtuple('Model', ['name', "Model"])
+
+def use_model(model):
```

Comment:
After a quick search, I have this library that looks actively maintained https://github.com/pytest-dev/pluggy/ (last updated 4 days ago and is used in pytest and tox and devpi), its worth a look as it may make development easier in the future.